lib/src/kbasesearchengine/search/ElasticIndexingStorage.java

```diff
@@ -547,6 +548,29 @@ public void flushIndexing(final ObjectTypeParsingRules rule) throws IOException
         return ImmutableMap.copyOf(ret);
     }
 
+    public boolean lookupParentId(final String type, final GUID guid) throws IOException {
```

Comment:
Javadoc would be nice here

Reply:
Maybe it should be called lookupParentIdExists? I would expect a lookupParentId to retrieve the parentID of the guid. 
The type has to be in a list of predetermined types or can it be an arbitrary type? 
The GUID is the guid of an updatedEvent right, or can it be any arbitraryGuid? The answer to those  might be able to go in the comments

Reply:
hasParentId that makes sense. what is an arbitraryGuid?

Reply:
like it could be an UpdateEventGuid vs a generic Guid